docs/de/contribute/README.md

```diff
@@ -0,0 +1,128 @@
+---
+home: false
+# layout: BlogHome
+icon: home
+title: Mitmachen
+sidebar: true
+tagLine: 
+# actions:
+#   - text: Demo
+#     link: https://stage.ocelot.social/
+---
+## Wie kannst du beitragen, mitmachen und unterstützen?
+
+Du kannst die freie Open-Source-Software *ocelot.social* und die aktiven Netzwerke auf vielfältige Art und Weise unterstützen, mitentwickeln oder mitprogrammieren.
+
+Klicke auf den Bereich, der dich interessiert:
+
+- [Spenden](#spenden)
+- [Unterstütze dein Netzwerk](#unterstutze-dein-netzwerk)
+- [Betreibe ein eigenes Netzwerk](#betreibe-ein-eigenes-netzwerk)
+- [Entwickle und unterstütze die Software](#entwickle-und-unterstutze-die-software)
+- [Weitere Ideen zum Mitmachen?](#weitere-ideen-zum-mitmachen)
+
+### Spenden
+
+Die Software des Netzwerks ist frei verfügbar – Betrieb, Unterhalt und Weiterentwicklung kosten hingegen Geld.
+
+Der Verein [busFaktor() e.V.](https://busfaktor.org/de/) koordiniert und fördert die Programmierung von *ocelot.social*.
+Unterstütze die Nachhaltigkeit  der Software und deren Netzwerke gerne mit einer Einmal- oder Dauerspende:
+
+<a href="https://busfaktor.org/de/spenden" target="_blank">
+  <Button class="donate-button">
+    Hier spenden
+  </Button>
+</a>
+
+### Unterstütze dein Netzwerk
```

Comment:
The purspose of this section is unclear.
An introduction that picks up the user from already being part of an ocelot.social-based network or operating their own network would be helpful.

Reply:
did a notice in the EPIC #49